C7Engine/EntryPoints/TurnHandling.cs

```diff
@@ -14,8 +14,7 @@ public static void EndTurn()
 
 			foreach (Player player in gameData.GetHumanPlayers()) {
 				foreach (MapUnit unit in player.units) {
-					Console.WriteLine($"{unit}, path length: {unit.path?.PathLength() ?? 0}");
-					if (unit.path?.PathLength() > 0) {
+					if ((unit.path?.PathLength() ?? -1) > 0) {
```

Comment:
This is probably modified due to the possibility of the Queue named `path` on line 8 of TilePath being null?  Which can probably only happen with TilePath.NONE?

If so, probably the right solution is to fix the defaults in TilePath - setting `path` to an empty (but not null) queue, and the `Tile destination` to Tile.NONE.  Then we shouldn't have to worry about null at all (and that also allows a bit of clean-up of null cases in TilePath).

(And yes, it looks like I didn't think of that when adding TilePath.  Oops)